src/test/java/courseWork/courseWork2/service/ExaminerServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package courseWork.courseWork2.service;
+
+
+import courseWork.courseWork2.exceptions.AmountExceeded;
+import courseWork.courseWork2.interfaces.QuestionService;
+import courseWork.courseWork2.question.Question;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ExaminerServiceImplTest {
+
+    @Mock
+    private JavaQuestionServiceImpl questionService;
```

Comment:
```suggestion
    private QuestionService questionService;
```
мокать надо по интерфейсу